tests/migrations/test_operations.py

```diff
@@ -2068,6 +2076,52 @@ def test_remove_field(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {"model_name": "Pony", "name": "pink"})
 
+    def test_remove_field_blocked(self):
+        """
+        Tests migration is blocked by dependent objects.
+        """
+        project_state = self.set_up_test_model("test_rmflb")
+        new_state = project_state.clone()
```

Comment:
What do we want to test here? is it a database behavior? I'm not sure it's worth doing, TBH.

It should be enough to test that `CASCADE` is not in a DDL operations for dropping a column.

Reply:
I agree, this test is quite complicated for the value it provides, and it's not ready for third-party backends as @felixxm noted.